screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -59,9 +59,9 @@ const SignUpScreen = ({navigation}) => {
     },
   });
 
-  const changeBackgroundColor = () =>
-    isChecked ? theme.colors.primary_green : theme.colors.light_gray;
-
+  const changeBackgroundColor = useMemo(() => {
+    return isChecked ? theme.colors.primary_green : theme.colors.light_gray;
```

Comment:
return directly withouth the word return

Reply:
Done !